spec/fixed_fields/comp_format_bad_008_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'nokogiri'
+require 'marc'
+require 'byebug'
+require 'marc_cleanup'
+
+RSpec.describe 'test  008 ' do
```

Comment:
In the `describe` string, the name of the method you're testing should be there in general. In this case, we are testing the `bad_008?` method.